plugins/VaccineAppointmentScanner/__init__.py

```diff
@@ -0,0 +1,119 @@
+from nextcord.ext import commands, tasks
+from Plugin import AutomataPlugin
+
+import requests
+
+## Mapping of major St. John's vaccine location IDs to location names
+LOCATIONS = {
+    949: "The Village Mall YELLOW",
+    943: "The Village Mall BLUE",
+    776: "FORMER Public Service Commission Building (50 Mundy Pond Road) (Ages <29)",
+    1668: "The Village Mall RED",
+    1669: "The Village Mall GREEN",
+    814: "Reid Community Centre (Location A) (Ages 30+)",
+    1672: "Reid Community Centre (Location B) (Ages 30+)",
+    1673: "Reid Community Centre (Location C) (Ages 30+)"
+}
+
+## The starting page for the vaccine booking system. Used to get a valid session cookie.
+START_URL = "https://portal.healthmyself.net/nleasternhealth/forms/P2E"
+
+## The first page for the vaccine booking process. Used to get a valid session cookie.
+BOOKING_URL = "https://portal.healthmyself.net/nleasternhealth/guest/booking/form/abec68ea-5a99-421b-b137-6e83cf7a3231"
+
+## The endpoint for accessing available appointment data.
+LOCATION_URL = "https://portal.healthmyself.net/nleasternhealth/guest/booking/type/5892/locations"
+
+class Appointments(AutomataPlugin):
+
+    def __init__(self, manifest, bot: commands.Bot):
+        super().__init__(manifest, bot)
+        self._binded_channels = []
+        self._previous_openings = []
+    
+    def cog_unload(self):
+        self.vaccineupdate.cancel()
+    
+    ## Retrieve appointment data from the booking system endpoint.
+    def get_appointment_data(self):
+        session = requests.Session()
+        session.headers.update({'referer': START_URL})
+        session.get(BOOKING_URL)
+
+        r = session.get(LOCATION_URL)
+        if (r.status_code != 200):
+            return None
+        else:
+            return r.json()
+
+    """Return the 1st shot/2nd shot/Booster COVID-19 vaccine locations with available appointments."""
```

Comment:
A small tweak by putting this inside the function (at the top) would also make it show when the a user looks for !help on what this command does